recipe/meta.yaml

```diff
@@ -1,15 +1,15 @@
 # Full credit goes to https://github.com/conda/conda-recipes for providing this recipe.
 # It has been subsequently adapted for automated building with conda-forge.
 
-{% set version = "3.5.2" %}
+{% set version = "3.5.3" %}
 
 package:
   name: matplotlib-suite
   version: {{ version }}
 
 source:
   url: https://github.com/matplotlib/matplotlib/archive/v{{ version }}.tar.gz
-  sha256: c81325d1b0d822ef75fd71d243f98478cdca1f800a838a2a1bc8e4e17b8df93c
+  sha256: 9d6ee7190fde3cb0c6285d462a0258c9735a72c7b532321a3983e05076aa41dc
 
 build:
   number: 0
```

Comment:
Now the minimal supported python version is **3.8,** see https://github.com/matplotlib/matplotlib/blob/318b2348f7127efc3e28aadebf762aae32b6343c/setup.py#L11
So we have to skip py<38 for matplotlib-base and matplotlib

Reply:
Conda-forge made changes in `requirements/host` for v3.5.3, see https://github.com/conda-forge/matplotlib-feedstock/commit/5cdeffcebbf5c4a05ef099936a0df900ebfc38c5. Could you verify if it's suitable for us?

Reply:
Not for the version in question though: https://github.com/matplotlib/matplotlib/blob/v3.5.3/setup.py#L11

When I download the source code, it also says 3.7.

Reply:
Ah, I see. Thanks!

Reply:
The minimum requirement for `python 3.8` seems to be for `Matplotlib 3.6` and above.
https://github.com/matplotlib/matplotlib/blob/318b2348f7127efc3e28aadebf762aae32b6343c/setup.py#L12

For `Matplotlib 3.5.3` the minnimum python version seems to be still `python 3.7`
https://github.com/matplotlib/matplotlib/blob/v3.5.3/setup.py#L11